velox/functions/sparksql/aggregates/tests/CMakeLists.txt

```diff
@@ -34,6 +35,7 @@ target_link_libraries(
   velox_functions_aggregates_test_lib
   velox_functions_spark_aggregates
   velox_hive_connector
+  velox_vector_fuzzer
```

Comment:
@mbasmanova I noticed that there's an omission here that hasn't been removed. I will remove it. Please help to reimport.